src/components/CameraPermissions/Primer/index.js

```diff
@@ -10,18 +10,18 @@ const Permissions = ({onNext, translate}) => (
   <div className={`${style.container} ${theme.fullHeightContainer}`}>
     <PageTitle title={translate('webcam_permissions.allow_access')} subTitle={translate('webcam_permissions.enable_webcam_for_selfie')} />
     <div className={`${theme.thickWrapper} ${style.bodyWrapper}`}>
-      <p className={style.instructions}>{translate('webcam_permissions.click_allow')}</p>
       <div className={style.image}>
-        <div className={style.graphic}>
-          <span className={style.allow}>{translate('webcam_permissions.allow')}</span>
-        </div>
+        <div className={style.graphic}></div>
```

Comment:
Do we still need this image class wrapper around `.graphic`?

Reply:
looks like, I am unsure though. Just made the changes that reflected in the desired design